dtfs-central-api/src/types/db-models/utilisation-reports.ts

```diff
@@ -40,3 +40,11 @@ export type UtilisationReport = Prettify<
     uploadedBy?: UploadedByUserDetails;
   }>
 >;
+
+export type UtilisationReportUploadDetails = Prettify<
+  Required<
+    Pick<UtilisationReport, 'azureFileInfo' | 'dateUploaded' | 'uploadedBy'> & {
+      status: Extract<UtilisationReportReconciliationStatus, 'PENDING_RECONCILIATION'>;
```

Comment:
This use of `Extract` doesn't seem to provide any value because the second argument can be anything. I don't get any auto-complete and can type and invalid value without error.

I think we could instead use this?
```
status: typeof UTILISATION_REPORT_RECONCILIATION_STATUS.PENDING_RECONCILIATION;
```

Reply:
That's strange because I did get autocomplete from using `Extract` _and_ I couldn't type any value that wasn't `'PENDING_RECONCILIATION'` for this field

Reply:
Discussed - seeing as we're using the constant everywhere anyway, we may as well keep using it here